### PR TITLE
Update Session helm chart

### PR DESCRIPTION
**Pull request by irunets**
Update Session helm chart

### CODE REVIEW

File: Charts/qtest-session/Chart.yaml [hunk 1]
```diff
@@ -18,14 +18,18 @@ dependencies:
...
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.5.4
+version: 1.6.0
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "2024.2"
+appVersion: 2024.2
```
Comment (pavel-spacil) [inline Charts/qtest-session/Chart.yaml:35]: This has to remain to be a string

File: Charts/qtest-session/Chart.yaml [hunk 2]
```diff
@@ -18,14 +18,18 @@ dependencies:
 maintainers:
   - name: byangtri
     email: b.yang@tricentis.com
+  - name: pjaladi21
```
Comment (pavel-spacil) [inline Charts/qtest-session/Chart.yaml:21]: Should be removed

File: Charts/qtest-session/values.yaml [hunk 3]
```diff
@@ -9,7 +9,7 @@ secrets:
 image:
   repository: qasymphony/sessions
   pullPolicy: IfNotPresent
-  tag: "2024.2"
+  tag: 2024.2
```
Comment (pavel-spacil) [inline Charts/qtest-session/values.yaml:12]: Use string here as well

File: Charts/qtest-session/templates/configmap.yaml [hunk 4]
```diff
@@ -30,4 +30,8 @@ data:
   qTestSessionPort: "{{ .Values.service.targetHttpsPort }}"
   {{- else }}
   qTestSessionPort: "{{ .Values.service.targetPort }}"
-  {{- end -}}
\ No newline at end of file
+  {{- end }}
+  qTestSessionNodeOptions: |-
```
Comment (tomas-kuty) [inline Charts/qtest-session/templates/configmap.yaml:34]: This is for 2024.4. It's not for 2024.3.

File: Charts/qtest-session/values.yaml [hunk 5]
```diff
@@ -80,6 +80,8 @@ qTestSession:
   qTestSessionDBSSLMountPath: "/etc/ssl"
   qTestSessionDBSSL: ""
   qTestSessionDBCRT: ""
+  # Array of cipher suites for SSL/TLS connection in OpenSSL format (e.g. ["TLS_AES_256_GCM_SHA384", "TLS_CHACHA20_POLY1305_SHA256"])
+  qTestSessionSSLCipherSuites: []
```
Comment (tomas-kuty) [inline Charts/qtest-session/values.yaml:84]: same, this is for 2024.4 only

**Review by tomas-kuty — CHANGES_REQUESTED** [review]
I'm gicing decline because some stuff is 2024.4 only and the rest is not needed.

**Pull request merged**